sudo/build.rs

```diff
@@ -222,6 +222,11 @@ fn main() -> io::Result<()> {
     Command::new("powershell")
         .arg("-NoProfile")
         .arg("-c")
+        .arg("Set-ExecutionPolicy")
+        .arg("Bypass")
+        .arg("-Scope")
+        .arg("Process")
+        .arg("-Force;")
```

Comment:
Huh, couldn't we just sign the script? I'm not sure who to ask for this internally, but I bet there's a chance we could do that.

Apropos, instead of a lot of args (even before your PR) I feel like anything past the `-c` should just be a single string, wrapped in single quotes. It's the command to be executed after all.

Reply:
PowerShell is optional and may not be present on the box, disabled, locked down, etc. and the user may not have permission to use that execution policy.

I would instead call out to `Get-ExecutionPolicy` and throw an error if the state is not a desirable one.